arrows/ocv/resection_camera.cxx

```diff
@@ -53,13 +57,15 @@ vital::config_block_sptr
 resection_camera
 ::get_configuration() const
 {
-  // get base config from base class
+  // Get base config from base class.
   vital::config_block_sptr config =
     vital::algo::resection_camera::get_configuration();
   config->set_value( "reproj_accuracy", d_->reproj_accuracy,
                      "desired re-projection positive accuracy" );
   config->set_value( "max_iterations", d_->max_iterations,
                      "maximum number of iterations to run optimization [1, INT_MAX]" );
+  // Get the camera configuration options.
+  d_->mvg::camera_options::get_configuration( config );
```

Comment:
This is unnecessary and feels odd. If the goal is clarity, maybe it would be better for `mvg::camera_options` to be a *member* of `priv` rather than a base?

Reply:
thanks for the note, I reflected that in the code